pkg/exec/local.go

```diff
@@ -66,9 +69,30 @@ func (cmd *LocalCmd) SetStderr(w io.Writer) Cmd {
 	return cmd
 }
 
-// Run runs
+// Run runs the command
+// If the returned error is non-nil, it should be of type *RunError
 func (cmd *LocalCmd) Run() error {
+	var out bytes.Buffer
+	// TODO(BenTheElder): adding bytes.Buffer to both multiwriters might need
+	// to be wrapped with a mutex
+	if cmd.Stdout != nil {
```

Comment:
can we have a case that cmd.Stdout is  = &out already and we have a multiwriter for the same output?